README.md

```diff
@@ -137,7 +137,7 @@ We're definitely not doing that. With react_on_rails, webpack is mainly generati
   foreman start -f Procfile.dev
   ```
 
-7. Visit [localhost:3000/hello_world](http://localhost:3000/hello_world)
+7. Visit [localhost:3000/hello_world](http://localhost:3000/hello_world) or [localhost:5000/hello_world](http://localhost:5000/hello_world) if the former doesn't work.
```

Comment:
Not sure we want to put that in. The generator is supposed to set the port, maybe.

This is really a puma configuration issue, and the port is listed lin the output.

@jbhatab would know about this one.